gamemodes/terrortown/gamemode/shared/lang/english.lua

```diff
@@ -1169,7 +1169,7 @@ L.shop_selected = "Selected {role}"
 L.shop_settings_desc = "Change the values to adapt Random Shop ConVars. Don't forget to save your changes!"
 
 L.f1_settings_changes_title = "Changes"
-L.f1_settings_hudswitcher_title = "HUD Switcher"
```

Comment:
This should still be called `HUD Switcher`, as the code refers to the HUD Editor as the position editor.
Also this will be reworked in version 1.0.

Reply:
Hmmm... look's like I got the point now.
HUD Editor is a part of the HUD Switcher, while the HUD Switcher is the left panel when you can choose what HUD you want to use or edit if it supports HUD Editor.

I will change it back to `HUD Switcher`.